tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -1,20 +1,23 @@
 #IB, NVLINK, or TCP
 CLUSTER_MODE=$1
+USERNAME=$(whoami)
 
 MAX_SYSTEM_MEMORY=$(free -m | awk '/^Mem:/{print $2}')M
 DEVICE_MEMORY_LIMIT="25GB"
 POOL_SIZE="30GB"
 
 # Fill in your environment name and conda path on each node
-CONDA_ENV_NAME="rapids-tpcxbb-20200706"
-CONDA_ENV_PATH="/raid/nicholasb/miniconda3/etc/profile.d/conda.sh"
+TPCX_BB_HOME="/home/$USERNAME/shared/tpcx-bb"
```

Comment:
If we're going to enforce this location by default, we should document it in the README